input_files/get-input-files.sh

```diff
@@ -4,4 +4,4 @@
 
 DIR=$(dirname $0)
 wget -O $DIR/input_files.tar.gz http://umich.edu/~mdolaboratory/repo_files/ADflow/adflow_input_files.tar.gz
-tar -xzf $DIR/input_files.tar.gz -C $DIR/../
+tar -xzf $DIR/input_files.tar.gz -C $DIR
```

Comment:
Seems that the new file is a tarbomb. I personally think we should refrain from that types of behavior.

Reply:
I thought it was safe because we were already opening it into a separate folder

Reply:
I think Eirikur is in general against tarbombs which is fair, although in this case the script specifically extracts to a designated directory.